Controller/SnapshotAdminController.php

```diff
@@ -46,7 +47,11 @@ public function createAction(Request $request = null)
 
         $snapshot->setPage($page);
 
-        $form = $this->createForm('sonata_page_create_snapshot', $snapshot);
+        $formType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix') ?
+            CreateSnapshotType::class :
+            'sonata_page_create_snapshot';
+
+        $form = $this->createForm($formType, $snapshot);
```

Comment:
We stopped supporting SF < 2.8 so you can go for:

```
$form = $this->createForm(CreateSnapshotType::class, $snapshot);
```

Reply:
yes indeed, changes were made before #877 was merged. But fixed now!